5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
@mfojtik , I think most of your questions are really questions for atomic developers. We are just trying to use what they provide. You are right that we should try to discuss our needs with them a bit and push them into "right" direction. @hhorak, maybe we should try more to discuss all of that with Atomic team and ask them for some solution for the problems @mfojtik pointed out?

> 7. Why this has to be privileged? We spend a lot of effort to make these images run under arbitrary UID, now we are saying that they should run as root?

As I understand Project Atomic, "atomic install" is designed to be run as root and it should execute single command which installs the docker image on the host system. Since single command is not enough for the creation of systemd service for the installed app or the volumes for the application, therefore the script is used.

That script is stored in the container itself and because the script must do all of that, it requires access to directories owned by root. Therefore it uses --privileged. I admit we could drop the rights in the script when calling assemble (or s2i in the future) and use the root only for things which strictly needs it.

Note that this is not something we have invented in RHSCL, it's documented in the Project Atomic [documentation](https://github.com/projectatomic/atomic/). Again, we could discuss with Project Atomic people if that's right or not, but for now we have decided to respect what Project Atomic offers and just use it...